### DISCUSSION TITLE
Additional security token?

### DISCUSSION THREAD

**Discussion by fonewiz** [ctwillie/expo-server-sdk-php#4] [Q&A] [answered]
Any way to send the security token using your SDK as outlined here?

Thanks.

**Comment by ctwillie** [comment, marked as answer]
@fonewiz Yes, make sure you have the latest version of the package by running ```composer update ctwillie/expo-server-sdk-php``` and call setAccessToken on the Expo instance:

```php

$expo = new Expo();

$expo->setAccessToken('xxx-xxx');
```
